lang-python/tzlocal/autobuild/defines

```diff
@@ -0,0 +1,9 @@
+PKGNAME=tzlocal
+PKGSEC=python
+PKGDEP="python-3"
+BUILDDEP="python-build python-installer wheel"
+PKGDES="tzinfo object for the local timezone"
```

Comment:
Python module to return an IANA time zone name for the local time or a tzinfo object